epimodel/exports/epidemics_org.py

```diff
@@ -134,4 +135,7 @@ def types_to_json(obj):
         return float(obj)
     if isinstance(obj, datetime.datetime):
         return obj.isoformat()
-    return obj
```

Comment:
Btw. this should be never done - that leads to circular reference errors: https://stackoverflow.com/questions/14249115/serializing-output-to-json-valueerror-circular-reference-detected